conda/recipes/versions.yaml

```diff
@@ -47,11 +47,11 @@ cupy_version:
 cython_version:
   - '>=0.29.17,<0.30'
 dask_version:
-  - '=2021.4.0'
+  - '=2021.5.0'
```

Comment:
Do we actually require this or can it be `>=2021.4.0,<=2021.5.0`?

Reply:
It can be `>=2021.4.0,<=2021.5.0`